lib/plugin/routes/public/teams.js

```diff
@@ -561,6 +561,45 @@ exports.register = function (server, options, next) {
     }
   })
 
+  server.route({
+    method: 'GET',
+    path: '/authorization/teams/{id}/nested',
+    handler: function (request, reply) {
+      const { organizationId } = request.udaru
+      const limit = request.query.limit || server.udaruConfig.get('authorization.defaultPageSize')
+      const page = request.query.page || 1
+      const { id } = request.params
+
+      request.udaruCore.teams.listNestedTeams({ organizationId, id, limit, page }, (err, data, total) => {
+        reply(err, err
+          ? null
+          : {
+            data,
+            total,
+            page: page,
+            limit: limit
+          })
+      })
+    },
+    config: {
+      validate: {
+        query: _.pick(validation.listNestedTeams, ['page', 'limit']),
+        params: _.pick(validation.listNestedTeams, ['id']),
+        headers
+      },
+      description: 'Fetch a nested team given its identifier',
+      notes: 'The GET /authorization/teams/{id}/nested endpoint returns a list of team data.\n',
+      tags: ['api', 'teams'],
+      plugins: {
+        auth: {
+          action: Action.ListNestedTeams,
+          getParams: (request) => ({ teamId: request.params.id })
+        }
+      },
+      response: { schema: swagger.List(swagger.NestedTeam).label('PagedTeams') }
```

Comment:
this will need to be refactored slightly after search is merged and it is rebased